### ISSUE TITLE
ColorYUV(analyze=true)

### ISSUE THREAD

**Issue by TbtBI** [AviSynth/AviSynthPlus#256]
```
ColorBars(pixel_type="yv12")
#propDelete("_ColorRange")
ColorYUV(analyze=true)
propshow()
```

Hi, if there is no frame prop for the color range, coloryuv(analyze=true) is returning a clip with wrong levels and wrong color range property. If there is frame prop, coloryuv(analyze=true) is returning a clip with correct levels and frame prop for the color range. You can test with the above script.

**Comment by pinterf** [comment]
Hi!, good observing. 
ColorYUV assumes PC range originally, as Avisynth Wiki says:
"[gamma] will operate on TV range instead of the default PC range when the input clip is known to be in TV range: .."

So when no information is given, the clip is assumed to be a full-range clip - ColorYUV speciality. If the _ColorRange property does not exist, it will be set to ColorYUV's default.

One would argue that when no conversion happens (no gamma is applied on the video or gamma is zero), the _ColorRange would stay undefined?

**Comment by TbtBI** [comment]
I never seen ColorYUV(analyze=true) to change the output in the period before the frame properties. I would expect the same behavior now too.

**Comment by pinterf** [comment]
True, will be fixed soon

**Comment by TbtBI** [comment]
Thanks.

**Comment by pinterf** [comment]
Hopefully fixed in 7b30966ef275e3ae2d5d7b0653d903da7c336335

**Comment by TbtBI** [comment]
It's ok now.
